### ISSUE TITLE
Feature: Add a stop button to journal

### ISSUE THREAD

**Issue by quozl** [sugarlabs/sugar#823]
Leaving the journal for the home view can be done with F3, or F6.  For new users, perhaps it would be handy to have a stop button which does the same thing.  An escape key might also do the same.  Downside is reduced journal search entry box size.

* [ ] discussion of user experience and design, leading to consensus,
* [ ] identify or add method for journal to activate home view,
* [ ] add stop button,
* [ ] add escape key handler.

**Comment by tony37** [comment]
The problem is that the Journal is treated as an activity. If it did not 
take the button of the current activity, the user can return by clicking 
on the current activity.
Since the Journal always has an active button, making it behave like an 
activty is unnecessary.

Tony

**Comment by quozl** [comment]
No, it doesn't always have an active button.  Some systems don't have <kbd>F5</kbd> key that works without pressing <kbd>Fn</kbd> first.  I'm working on such a system now.

**Comment by Devil-Echo** [comment]
I am trying to make changes in the live build - /usr/src/sugar-activities/.. / journalactivity.py file but the changes could not be saved to the disk, giving permission error.
Plz help
